From 61df26328ed819e4f220760a98ce31529c4ec609 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <ignatenko@redhat.com>
Date: Tue, 23 Aug 2016 15:17:09 +0200
Subject: [PATCH] cli/repolist: fix showing repository name with 'disabled' arg
 (RhBug:1369459)

References: https://bugzilla.redhat.com/show_bug.cgi?id=1369459
Signed-off-by: Igor Gnatenko <ignatenko@redhat.com>

Closes: #579
Approved by: ignatenkobrain

(cherry picked from commit f41c4ebc5467f9f7b161545a5a416f0778f2ed96)
---
 dnf/cli/commands/repolist.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dnf/cli/commands/repolist.py b/dnf/cli/commands/repolist.py
index 110db4d..0cf67f1 100644
--- a/dnf/cli/commands/repolist.py
+++ b/dnf/cli/commands/repolist.py
@@ -260,7 +260,7 @@ class RepoListCommand(commands.Command):
             for (rid, rname, (ui_enabled, ui_endis_wid), ui_num) in cols:
                 if arg == 'disabled': # Don't output a status column.
                     print("%s %s" % (fill_exact_width(rid, id_len),
-                                     fill_exact_width(rname, nm_len, -nm_len)))
+                                     fill_exact_width(rname, nm_len, nm_len)))
                     continue
 
                 if ui_num:
-- 
2.9.3

